servicetalk-http-netty/src/main/java/io/servicetalk/http/netty/DefaultHttpLoadBalancerFactory.java

```diff
@@ -214,4 +278,188 @@ public String toString() {
             return delegate.toString();
         }
     }
+
+    private static final class DefaultHttpLoadBalancedConnection
```

Comment:
This new class should also have `toString()` impl. Otherwise, it removes netty's channel info from logs.
We have tests that verify it for the regular path, they didn't trigger assertions bcz they don't go through new lb.

Reply:
Another option could be to add a new pkg-private ctor for `HttpLoadBalancerFactory.DefaultFilterableStreamingHttpLoadBalancedConnection` instead of creating a new class.

Reply:
We will need to override the concurrency controller callbacks, and have null checks to keep old/new behavior. I think this looks cleaner and its easier to remove when we clean the deprecation.